resources/functionMap.php

```diff
@@ -87,12 +87,12 @@
 'AMQPConnection::disconnect' => ['void'],
 'AMQPConnection::getHost' => ['string'],
 'AMQPConnection::getLogin' => ['string'],
-'AMQPConnection::getMaxChannels' => ['int'],
+'AMQPConnection::getMaxChannels' => ['int<1,65535>'],
```

Comment:
Similar to above, a channel of 0 is invalid, despite being a 16bit unsigned int.